### PR TITLE
[fuchsia] Remove old, pre-undercoat code (re-land)

### PR DESCRIPTION
**Pull request by eepeep**
Now that undercoat is the sole interface between ClusterFuzz and
Fuchsia, support for pre-undercoat builds can be removed. This will
potentially cause failures during regression testing against older
builds, but this should be uncommon and not block any other tasks.

### CODE REVIEW

**Comment by eepeep** [comment]
cc @oliverchang 

This is a followup to #2559, removing the offending reference (since it was a no-op function anyways.)  It's a separate PR because I couldn't figure out how to push to #2554 now that it's closed.

Apologies for missing the stray reference!

**Review by oliverchang — APPROVED** [review]
Thanks! I'll wait until Monday to merge :)

**Pull request merged**
